### ISSUE TITLE
workingDirectory is defined twice in TerraformCLI

### ISSUE THREAD

**Issue by DarkestOfNights** [charleszipp/azure-pipelines-tasks-terraform#17]
In TerraformCLI/task.json, the workingDirectory input variable is defined twice with different labels. If one is populated, the other input becomes populated at the same time, so you won't have the possibility of one overwriting the other, though it could lead to confusion.

``` json
53:   {
      "name": "workingDirectory",
      "type": "filePath",
      "label": "Configuration Directory",
      "defaultValue" : "$(System.DefaultWorkingDirectory)",
      "helpMarkDown": "Terraform configuration directory. If empty, the root of the repository (build) or artifacts (release) is used, which is $(System.DefaultWorkingDirectory)",
      "required": "false"
    },
   ...
81:    {
      "name": "workingDirectory",
      "type": "filePath",
      "label": "Working Directory",
      "helpMarkDown": "Current working directory where the command is run. If empty, the root of the repository (build) or artifacts (release) is used, which is $(System.DefaultWorkingDirectory)",
      "required": "false"
    },
```

**Comment by charleszipp** [comment]
@DarkestOfNights thanks for pointing this out! I can get this corrected.

**Comment by charleszipp** [comment]
released with 0.3.1

cc @DarkestOfNights
